### DISCUSSION TITLE
Readme for Dummies?

### DISCUSSION THREAD

**Discussion by Pierre-de-la-Verre** [yWorks/svg2pdf.js#299] [Q&A]
Hi
is there a "README" for Dummies?

For me, I don't know how to use "npm" - inside Windows cmd? Inside the console of my browser? Any requirements on user rights or software?
What's about the "import ..." stuff? Where and how to use? Use it once (as "setup") or always ....??

Thanks for help!

**Comment by yGuy** [comment]
This is a software library, not a tool to be used by end-users. It cannot be used without programming. As such, there is no readme for non-programmers. Sorry!

If you are looking for a tool to convert SVG to PDFs, I suggest you use your browser - just open the SVG and print it. The playground we have for this project would also provide you with this capability, however I don't see any advantages over using the browser, directly. Only disadvantages.
